src/Routes/Categories.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react'
+
+export default class In  extends Component {
+  render() {
+    return (
+      <h2>In construction</h2>
+    )
+  }
```

Comment:
- According to the project's requirement, The categories component should display only a button with the text "Check status". In yours, you have 'In construction'. Can we kindly fix that? 🌸